trezor/attest/core/v2.6.0/attest.B56A8EF129DEEED6

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP SIGNED MESSAGE-----
+Hash: SHA256
+
+verify.sh Version:       0.2
+Intended Build Version:  core/v2.6.0
+build-docker.sh Version: core/v2.6.0
+Build Command:           build-docker.sh --skip-legacy core/v2.6.0
+Bootloader Build?:       0
+DD Zero Sig Options:     bs=1 seek=5567 count=65 conv=notrunc status=none
+
+c33e336869964cfb1ef193195894e8b6667955b4ea3044558c380b1787168e38  trezor-2.6.0.bin
+e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  repo/build/core/firmware/firmware.bin
```

Comment:
The `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855` checksum is a null-file checksum.  The build must have had a catastrophic failure on either the docker container or the build itself.  I should put a file-size check in there to alert.  Normally if the checksums don't match, I just dump it in failed.